### DISCUSSION TITLE
LazyVim website colour theme

### DISCUSSION THREAD

**Discussion by antogilbert** [LazyVim/LazyVim#797] [Q&A]
Hello, I was browsing the configuration of LazyVim and I noticed that the website has a very nice dark purple colour scheme. I thought that would be the tokyonight theme but the one on the website is slightly different, since tokyonight has very bright orange and cyan accents, whereas the website colourscheme has light magenta tones (which I find easier on the eyes). Does anyone know what colour scheme is used on the website and if there is a neovim equivalent?

This is a snippet of code on the website: https://www.lazyvim.org/configuration#default-settings

Thanks!

**Comment by folke** [comment]
I created the theme myself. See here https://github.com/folke/tokyonight.nvim/tree/main/extras/prism

The one on the website is the tokyonight moon style. To use it in Neovim, set `style="moon"` or do `:colorscheme tokyonight-moon`
